### PR TITLE
decent version :)

### PR DESCRIPTION
**Pull request by johmathe**

### CODE REVIEW

File: pipeline.py [hunk 1]
```diff
@@ -96,21 +94,35 @@ def process_file(path, output):
...
     if mean > 1.0:
         print('mean too high: %s' % mean)
         return
-    array -= mean
-    for k in range(array.shape[2]):
+
+    processed_file = get_tempfile_name()
+    os.system('/usr/lib/ants/N4BiasFieldCorrection -i %s -o %s -s 6' %
+              (path, processed_file))
+    #os.system('cp %s %s' %
```
Comment (ninamiolane) [inline pipeline.py]: leftover comment

File: pipeline.py [hunk 2]
```diff
@@ -96,21 +94,35 @@ def process_file(path, output):
...
     mean = np.mean(array.reshape(-1))
     # HACK Alert - This is a way to check if the backgound is a white noise.
     if mean > 1.0:
         print('mean too high: %s' % mean)
         return
-    array -= mean
-    for k in range(array.shape[2]):
+
+    processed_file = get_tempfile_name()
+    os.system('/usr/lib/ants/N4BiasFieldCorrection -i %s -o %s -s 6' %
```
Comment (ninamiolane) [inline pipeline.py]: can we add the skull-stripping with a similar line?

File: pipeline.py [hunk 3]
```diff
@@ -96,21 +94,35 @@ def process_file(path, output):
...
+              (path, processed_file))
+    #os.system('cp %s %s' %
+    #          (path, processed_file))
+    img = nibabel.load(processed_file)
+
+    array = img.get_fdata()
+    array = np.nan_to_num(array)
+    std = np.std(array.reshape(-1))
+    # No centering because we're using cross-entropy loss.
+    # Another HACK ALERT - statisticians please intervene.
```
Comment (ninamiolane) [inline pipeline.py]: xD

File: pipeline.py (hunk 2 above)
Comment (johmathe) [reply pipeline.py]: I think so but skull stripping might take ages. I can add a todo

**Pull request merged**